tests/protocols/guidance_test.py

```diff
@@ -0,0 +1,178 @@
+from client.protocols import unlock
+from client.protocols import guidance
+from client.sources.common import models
+import mock
+import unittest
+import os
+
+
+class GuidanceProtocolTest(unittest.TestCase):
+	GUIDANCE_DIRECTORY = "demo/guidance_test/"
```

Comment:
These are tabs (you can see them when you highlight). Please change all tabs to 4 spaces. You should probably configure your editor to indent with spaces by default, too.